packages/terraform-modules-public/aws/README.md

```diff
@@ -0,0 +1,28 @@
+# Terraform Celo Validator Stack
+
+## Overview
+
+[Terraform](https://www.terraform.io) is a tool by Hashicorp that allows developers to treat _"infrastructure as code"_, easying the management and repeatibility of the
+infrastructure.
+Infrastructure and all kind of cloud resources are defined in modules, and Terraform creates/changes/destroys when changes are applied.
+
+Inside the [testnet](./testnet) folder you will find a module (and submodules) to create the setup for running a Celo Validator on AWS. The next logic resources can be created:
+
+- `vpc` module for setting up a VPC with a public and private subnet on multiple availability zones. Validator nodes go in the private subnet while bastion, proxy, and attestation nodes go in the public subnet.
+- `bastion` module for an SSH bastion node. For security purposes, this is the only node that accepts external SSH traffic. All other nodes only accept SSH from the bastion.
+- `proxy` module for creating a Proxy connected to a validator
+- `validator` module for deploying a Validator
+- `attestation-service` for deploying the Attestation Service (https://docs.celo.org/getting-started/baklava-testnet/running-a-validator#running-the-attestation-service). Each attestation service will connect to the same multi-AZ RDS DB instance. Only Twilio is supported at this time.
+
+## Operating System
+
+All nodes run on the Ubuntu LTS 18.04 AMI. Running `terraform apply` will select the latest available AMI.
+
+## Hardening & Security
+
+Celo nodes will get provisioned with some recommended security settings from Celo's Security Audit team. Most of this hardening is done in `install-base.sh` and `final-hardening.sh`. 
+
+## Requirements
+
+Inside the [example](./example) folder you can find an example tf to use the module. We recommend you to use that tf as base file for your deployment, modifying the account variables used for your convenience.
```

Comment:
In an effort to emphasize documentation as something that is also decentralized, could you reword the places where "we" is used? eg "We recommend" -> "It's recommended"

Reply:
Will do

Reply:
9399d1a6f0f14179c97c8ecf6f3f248a080cc158